crates/bevy_animation/Cargo.toml

```diff
@@ -14,10 +14,15 @@ bevy_app = { path = "../bevy_app", version = "0.12.0-dev" }
 bevy_asset = { path = "../bevy_asset", version = "0.12.0-dev" }
 bevy_core = { path = "../bevy_core", version = "0.12.0-dev" }
 bevy_math = { path = "../bevy_math", version = "0.12.0-dev" }
-bevy_reflect = { path = "../bevy_reflect", version = "0.12.0-dev", features = ["bevy"] }
+bevy_reflect = { path = "../bevy_reflect", version = "0.12.0-dev", features = [
```

Comment:
were all of these reformats really intended?

Reply:
not really, it was just the formatting of the Even Better TOML extension in vscode